net.sourceforge.tomatoes.IHaveNoTomatoes.json

```diff
@@ -0,0 +1,111 @@
+{
+  "app-id": "net.sourceforge.tomatoes.IHaveNoTomatoes",
+  "default-branch": "stable",
```

Comment:
```suggestion
```

Reply:
OK. Is there something I overlooked in the instructions explaining why I was wrong in doing that?

Reply:
yeah the "instructions" needs to be updated. 

The branch is set build the build bot automatically.